covid19-dashboard/src/Content.json

```diff
@@ -0,0 +1,57 @@
+{
+  "cumulativePanel": [{
+    "title": "Total Cases",
+    "dataKey": "cumulativeCases",
+    "color": "--dc-red-strong"
```

Comment:
I have a Color() function in Utils that takes in a name or hex.

If defined in root, it converts to hex. Otherwise, it will use that color. So in the above example, a hex would also work.

Is this correct?

Color("--dc-red-strong") -> "#660000"
Color("#003254") -> "#003254"